e2e/bzlmod/.bazelrc

```diff
@@ -1,4 +1,4 @@
-# import common bazelrc shared with e2e workspaces
-import %workspace%/../../.bazelrc.common
+# NB: we don't import common bazelrc shared with e2e workspaces
```

Comment:
Why? What caused this in this PR?

Reply:
I intentionally changed this as the regression test. It was erroring because the transpiler flag wasn't set. Now it's green because there's an explicit "tsc"